boards/native/Makefile.include

```diff
@@ -72,7 +72,7 @@ export LINKFLAGS += -ffunction-sections
 ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
   PORT ?= tap0
 else
-  PORT =
+  PORT ?=
```

Comment:
Does it make sense to override an empty string (which an unset variable evaluates to in `make`) with an empty string? Why not just remove the else case all together?

Reply:
I thought someone put it there for some (unkown to me) reason, so I figured I leave it. So should I remove it?

Reply:
The only reason I could imagine is to unset `PORT` if it is set, but `netdev_default`/`gnrc_netdev_default` is not used. However, your change removes that behavior, so it makes more sense to just remove it altogether.